catroid/src/main/java/org/catrobat/catroid/common/defaultprojectcreators/ArDroneProjectCreator.java

```diff
@@ -66,6 +68,14 @@ public Project createDefaultProject(String projectName, Context context, boolean
 			throw new IllegalArgumentException("Project with name '" + projectName + "' already exists!");
 		}
 
+		Project defaultDroneProject = new Project(context, projectName, landscapeMode);
+		defaultDroneProject.setDeviceData(context); // density anywhere here
```

Comment:
what does the comment mean?

Reply:
no Idea, therefor I didn't delete it :(

Reply:
then it doesnt help anyways, i also have no clue what thats supposed to mean. so might aswell delete it.

Reply:
done